acm/scripts/prepare-olm.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+#
+
+set -e
+
+release="v0.27.0"
```

Comment:
```suggestion
release="v0.28.0"
```
Just curious if we chose this version intentionally when compared to latest